config/examples/Creality/Ender-3/BigTreeTech SKR E3-DIP V1.1/Configuration_adv.h

```diff
@@ -2578,7 +2578,7 @@
    * Beta feature!
    * Create a 50/50 square wave step pulse optimal for stepper drivers.
    */
-  #define SQUARE_WAVE_STEPPING
+  //#define SQUARE_WAVE_STEPPING
```

Comment:
This should certainly be left enabled.
It isn’t really noticeable at the moment, but I think it should be the default where possible. It simple does nothing if the drivers don’t support it.

Reply:
It was disabled by default and has "beta" status. Am I wrong that this parameter is meaningful only for SpreadCycle mode? And by default the extruder driver mode is stealthChop. Anyway I don't object to the option enabling.